pyproject.toml

```diff
@@ -1,6 +1,59 @@
+# SPDX-FileCopyrightText: 2022 Tanguy Fardet
+# SPDX-License-Identifier: CC0-1.0
+
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pynsee"
+authors = [
+  { name="Hadrien Leclerc", email="leclerc.hadrien@gmail.com" },
+  { name="Lino Galiana" },
```

Comment:
Note : I just sent an email to Lino to validate setting his professionnal email.